include/print_log.h

```diff
@@ -42,6 +42,10 @@
 
 #include <stdio.h>
 
+#ifndef LOG_LEVEL
+#define LOG_LEVEL 0x6
```

Comment:
I'd just move this below these defines and 
#ifndef LOG_LEVEL
#define LOG_LEVEL LOG_INFO
#endif

Reply:
Sure.